content/lua/concepts/strings/terms/lower/lower.md

```diff
@@ -0,0 +1,31 @@
+---
+Title: 'lower' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
```

Comment:
```suggestion
Title: 'lower()' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
```